website/client/src/pages/settings/settingRows/loginMethods.vue

```diff
@@ -0,0 +1,186 @@
+<template>
+  <fragment>
+    <tr
+      v-for="network in SOCIAL_AUTH_NETWORKS"
+      :key="network.key"
+    >
+      <td class="settings-label">
+        <div class="network-icon-with-label">
+          <span
+            :class="'svg-icon icon-16 social-icon ' + network.key"
+            v-html="icons[network.key]"
+          ></span>
+
+          <span class="ml-75"> {{ network.name }}</span>
+        </div>
+      </td>
+      <td class="settings-value">
+        <div
+          v-if="isConnected(network.key)"
+          class="connected-pill"
+        >
+          {{ $t('connected') }}
+        </div>
+      </td>
+      <td class="settings-button">
+        <a
+          v-if="allowedToConnect(network.key)"
+          class="edit-link"
+          @click.prevent="socialAuth(network.key, user)"
+        >
+          {{ $t('connect') }}
+        </a>
+        <a
+          v-if="allowedToRemove(network.key)"
+          class="remove-link"
+          @click.prevent="deleteSocialAuth(network)"
+        >
+          {{ $t('remove') }}
+        </a>
+      </td>
+    </tr>
+  </fragment>
+</template>
+
+<script>
+import axios from 'axios';
+import hello from 'hellojs';
+import { buildAppleAuthUrl } from '@/libs/auth';
+import { mapState } from '@/libs/store';
+import { SUPPORTED_SOCIAL_NETWORKS } from '../../../../../common/script/constants';
+import googleIcon from '@/assets/svg/google.svg';
+import appleIcon from '@/assets/svg/apple_black.svg';
+
+export default {
+  name: 'LoginMethods',
+  data () {
+    return {
+      SOCIAL_AUTH_NETWORKS: [],
+      // Made available by the server as a script
+      localAuth: {
+        password: '',
+        confirmPassword: '',
+      },
+      icons: Object.freeze({
+        google: googleIcon,
+        apple: appleIcon,
+      }),
+    };
+  },
+  computed: {
+    ...mapState({
+      user: 'user.data',
+      content: 'content',
+    }),
+  },
+  mounted () {
+    this.SOCIAL_AUTH_NETWORKS = SUPPORTED_SOCIAL_NETWORKS;
+
+    this.$store.dispatch('common:setTitle', {
+      section: this.$t('settings'),
+    });
+
+    hello.init({
+      facebook: process.env.FACEBOOK_KEY, // eslint-disable-line no-process-env
```

Comment:
We don't support Facebook anymore, do we still need this?

Reply:
removed